cmd/info/info.go

```diff
@@ -0,0 +1,69 @@
+package info
+
+import (
+	"github.com/CircleCI-Public/circleci-cli/api/info"
+	"github.com/CircleCI-Public/circleci-cli/settings"
+	"github.com/olekukonko/tablewriter"
+
+	"github.com/spf13/cobra"
+)
+
+//infoOptions info command options
+type infoOptions struct {
+	cfg       *settings.Config
+	validator validator
+}
+
+type validator func(cmd *cobra.Command, args []string) error
+
+// NewInfoCommand information cobra command creation
+func NewInfoCommand(config *settings.Config, preRunE validator) *cobra.Command {
```

Comment:
we are passing in `validator` to `NewInfoCommand` in `cmd/root.go` but the arg is not used in this function, can we remove it?

Reply:
gotcha, just wasn't sure 👍 
may want to change `preRunE` to `_` to appease the linter 😁